src/main.ts

```diff
@@ -169,7 +169,7 @@ async function deactivateRevision(params: any) {
     revisionName
   )
   if(deactiveRevision.active) {
-    console.log("Deactivation Step Error");
+    throw new Error(`The revision ${revisionName} under container app ${containerAppName} can't be deactivated. Check the Azure Portal for details.`);
   } else {
```

Comment:
The `else` block is not needed because the the process jumps into a `catch` block if an exception is thrown at L172.

Reply:
The `else` block is for a `Succeeded` case.

Reply:
FYI. The following is same as code and more simple.

```javascript
if (deactiveRevision.active) throw new Error (...);

console.log("Deactivation Step Succeeded");
```